README.md

```diff
@@ -30,7 +30,10 @@ cerberus:
         -    openshift-kube-scheduler
         -    openshift-ingress
         -    openshift-sdn
-        -    openshift-ovn-kubernetes
+       #-    openshift-ovn-kubernetes                    # Enable when OVNKubernetes is used as the SDN. Disable openshift-sdn when that is the case
+
+       #-    openshift-node-problem-detector             # Enable only when node problem detector is deployed, check README for more details
```

Comment:
Thinking if we should remove the option/ns from the config since there is no need to monitor the status of the node-problem-detector pods/namespace, we just need to monitor the conditions it adds to the node when it's enabled/running. Thoughts?